modules/@apostrophecms/schema/ui/apos/logic/AposInputSlug.js

```diff
@@ -55,7 +55,6 @@ export default {
       // We are usually interested in followingValue.title, but a
       // secondary slug field could be configured to watch
       // one or more other fields
-      deep: true,
```

Comment:
must be removed here, `followingValues` being recomputed all the time (should be improved in the future..).
We just need to compare values to see if a value changed.